### ISSUE TITLE
Unsupported swagger/OpenAPI version: 3.2

### ISSUE THREAD

**Issue by sarkistlt** [Mermade/widdershins#87]
is it something that will be implemented soon?

Thanks.

**Comment by MikeRalphson** [comment]
I'm not sure I understand. The latest version of the OpenAPI specification is v3.0.1.

**Comment by sarkistlt** [comment]
that's the error I'm getting when trying to run this:

```js
const converter = require('widdershins');
const apiObj = require('./swagger.json');
const options = {}; // defaults shown
options.codeSamples = true;
//options.language_tabs = [];
//options.loadedFrom = sourceUrl;
//options.user_templates = './user_templates';
options.templateCallback = function (templateName, stage, data) { return data };
options.theme = 'darkula';
options.search = true;
options.sample = true; // set false by --raw
options.schema = true; // set false by --noschema
options.discovery = false;
options.includes = [];
options.summary = false;
options.headings = 2;

converter.convert(apiObj, options, function (err, str) {
  console.error('err', err);
  console.log('str', str);
});
```
`swagger.json` generated by https://github.com/feathersjs-ecosystem/feathers-swagger

**Comment by MikeRalphson** [comment]
Feathers-swagger is confusing the version number of swagger-ui (3.2) with the version of the specification (2.0).

It appears https://github.com/feathersjs-ecosystem/feathers-swagger/commit/b298f9c77a28062d1a068b61e28afc0bc14909c4 is the culprit.

**Comment by sarkistlt** [comment]
I see, but this error coming from widdershins, when trying to parse swagger 2, but this module says that it support Swagger 2.0, so is it something wrong with JSON? it works fine with swagger ui

**Comment by sarkistlt** [comment]
btw, it works fine with https://github.com/lavkumarv/swagger-to-slate

**Comment by sarkistlt** [comment]
works, just need to fix version in json from 3.2 to 2.0, thanks!
